lib/routes/emails.js

```diff
@@ -452,29 +451,11 @@ module.exports = (log, db, mailer, config, customs, push) => {
                       })
                     ])
                   })
-                  .then(() => {
-                    if (reminder === 'first' || reminder === 'second') {
```

Comment:
I wonder if it makes sense to keep this so we can track when last remaining reminders have been used?

Reply:
Ah, that is a good point. Will put back.

Reply:
Wait, no I won't. There aren't any remaining reminders, because it's been disabled for so long. See:

https://sql.telemetry.mozilla.org/queries/50754/source